packages/create/README.md

```diff
@@ -148,6 +149,15 @@ Calling `sap-ux change inbound` allows replacing the Inbound FLP configurations
 sap-ux change inbound [path]
 ```
 
+## sap-ux convert
+Executing `sap-ux convert` converts an app to a new feature.
+
+### preview
+Executing `sap-ux convert preview-config` in the root folder of an app will convert the respective app to the preview with virtual files. It will use the configuration from the scripts in the `package.json` file to adjust the UI5 configuration YAML files accordingly. The obsolete JS and TS sources will be deleted and the HTML files previously used for the preview will be renamed to `*_old.html`.
+```sh
+sap-ux convert preview [path]
+```
+
 ## sap-ux remove
 Calling `sap-ux remove` allows removing a feature to a project.
```

Comment:
```suggestion
Calling `sap-ux remove` allows removing a feature from a project.
```

Reply:
Done in f41acea6cdc7bcf341fd9c72b7a4306e0012a3b5